cassandra_nodetool/tests/conftest.py

```diff
@@ -47,7 +41,7 @@ def get_container_ip(container_id_or_name):
         '-f', '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}', container_id_or_name
     ]
 
-    return subprocess.check_output(args).rstrip('\r\n')
+    return subprocess.check_output(args).strip('\r\n')
```

Comment:
I think ofek meant you should also remove the `\r\n` as by default the `strip` method removes that.

Reply:
Yeah of course! I reapplied the logic that was there before without thinking :\